blivet/util.py

```diff
@@ -226,6 +226,18 @@ def get_sysfs_path_by_name(dev_node, class_name="block"):
         raise RuntimeError("get_sysfs_path_by_name: Could not find sysfs path "
                            "for '%s' (it is not at '%s')" % (dev_node, dev_path))
 
+def get_cow_sysfs_path(dev_path, dev_sysfsPath):
+    """ Return sysfs path of cow device for a given lvm snapshot.
```

Comment:
*"for a given device"* would be better as it is more generic (as this function potentially is) and doesn't create a confusion by *"lvm snapshot"* suddenly appearing here for no obvious reason.